crypto/secp256k1.go

```diff
@@ -117,6 +121,15 @@ func (key *Secp256k1PrivateKey) ToHex() string {
 	return util.BytesToHex(key.Bytes())
 }
 
+// ToAIP80 formats the private key to AIP-80 compliant string
+func (key *Secp256k1PrivateKey) ToAIP80() string {
+	var res, err = FormatPrivateKey(key.ToHex(), PrivateKeyVariant("secp256k1"))
+	if err != nil {
+		panic(err)
+	}
+	return res
+}
```

Comment:
Yeah, it should ideally output the error.

Maybe I was naive on my functionality earlier, or we need to make it so there are two implementations, one that won't fail, and one that will.

Basically because you're using a function that's used for 2 cases in a specific case.